docs/source/install_instructions.rst

```diff
@@ -5,18 +5,13 @@ Installation Instructions
     TorchCodec is only available on Linux for now. We plan to support other
     platforms in the future.
 
-Installing torchcodec should be as simple as:
+First, install the latest stable version of PyTorch following the `official
+instructions <https://pytorch.org/get-started/locally/>`_. TorchCodec requires
+`PyTorch 2.4 <https://pytorch.org/docs/2.4/>`_.
 
-.. code:: bash
-
-    pip install torchcodec
-
-You will need a working PyTorch installation, which you can install following
-the `official instructions <https://pytorch.org/get-started/locally/>`_.
-
-You will also need FFmpeg installed on your system, and TorchCodec decoding
+You will also need FFmpeg installed on your system. TorchCodec's decoding
 capabilities are determined by your underlying FFmpeg installation. There are
-different options to install FFmpeg e.g.:
+different options to install FFmpeg, e.g.:
```

Comment:
Instead of converting a list to English, it would be more compact and easy to read something like this:

TorchCodec installation steps:

1. Install pytorch using ...
2. Install ffmpeg using ...
3. Install torchcodec using `pip install torchcodec`

Seems more compact and readable.

Reply:
I did that initially, then reverted it because it felt like too much ceremony. I had tried sub-subsections. I'm not sure how well it will work having code blocks in numbered lists. Let me try and see how it renders.